app/tasks/table_source_assets.py

```diff
@@ -188,6 +189,112 @@ async def table_source_asset(
     return log
 
 
+async def append_table_source_asset(
+    dataset: str, version: str, asset_id: UUID, input_data: Dict[str, Any],
+) -> ChangeLog:
+    creation_options = TableSourceCreationOptions(**input_data["creation_options"])  # TODO get from row
+    source_uris: List[str] = input_data["source_uri"]
+
+    field_metadata = await _get_field_metadata(dataset, version)
+    field_names = f"({','.join([field['field_name_'] for field in field_metadata if field['is_feature_info']])})"
```

Comment:
should be field['field_name'].

field_name is a reserved name in pydantic and we need to add the underscore when defining the property. However, since we declared the alias, we can use the alias in the dictionary. Just make sure that you export pydantic models to dict using my_model.dict(by_alias=True)

Reply:
Make sure to update from develop. There was a recent update which addressed exactly this issue and some bugs around it

Reply:
got rid of this